### PR TITLE
Refine tag editor - part2

### PR DESCRIPTION
**Pull request by jasondoong**

### CODE REVIEW

**Comment by jasondoong** [comment]
這個PR把編輯標籤dialog剩下沒做的做完。

File: src/components/tagSettings.jsx [hunk 1]
```diff
@@ -50,91 +23,64 @@ const nameTemplate = (tag) => {
...
+  const operationsTemplate = (tag) => {
+    const [showEditDialog, setShowEditDialog] = useState(false);
+    const [showDeleteDialog, setShowDeleteDialog] = useState(false);
+    return (
+      <div className="flex items-center">
+        <Button
+          className="p-button-text p-button-raised"
+          onClick={() => setShowEditDialog(true)}
+        >
+          編輯
```
Comment (TungYing) [inline src/components/tagSettings.jsx:45]: it can add to i18n file

File: src/components/tagSettings.jsx [hunk 2]
```diff
@@ -50,91 +23,64 @@ const nameTemplate = (tag) => {
...
+  };
 
   return (
-    <div className="block">
-      <div className="flex justify-between my-6">
+    <>
+      <div className="flex justify-between mb-4">
         <p className="text-xl font-bold">標籤列表</p>
-        <Button className="p-button-outlined text-xs">建立標籤</Button>
+        <Button className="p-button-outlined text-xs" onClick={() => setShowCreateDialog(true)}>建立標籤</Button>
```
Comment (TungYing) [inline src/components/tagSettings.jsx:72]: same issue

**Comment by jasondoong** [comment]
@TungYing updated.

**Review by TungYing — APPROVED** [review]
goods.

**Pull request merged**